README.md

```diff
@@ -181,6 +181,14 @@ function Bar (N : Natural) return Natural is
       Convention => C;
 ```
 
+```Ada
+type Enum is
+   (Very_Very_Very_Very_Long_Literal_1 => 1,
+    Very_Very_Very_Very_Long_Literal_2 => 2,
+    Very_Very_Very_Very_Long_Literal_3 => 3)
+with Aspect => Condition;
+```
```

Comment:
Does this style only apply for very long literals or for all cases? Do we want to allow single line enums?

```Ada
type Enum is (A => 1, B => 2, C => 3) with Aspect => Condition;
```

Reply:
I added a comment to this in #24 since there are more implications to this.

Reply:
Fixed in https://github.com/Componolit/ada-style/pull/25/commits/2d9a52028ec9141402319880d6242abc794eedf0.